### PR TITLE
modify Eng comma

### PR DESCRIPTION
**Pull request by yang131313**
modify Eng comma

### CODE REVIEW

File: docs/api/paddle/Model_cn.rst [hunk 1]
```diff
@@ -13,12 +13,12 @@ Model
 :::::::::
 
     - **network** (paddle.nn.Layer): 是 ``paddle.nn.Layer`` 的一个实例
-    - **inputs** (InputSpec|list|dict|None, 可选):  ``network`` 的输入，可以是 ``InputSpec`` 的实例，或者是一个 ``InputSpec`` 的 ``list`` ,或者是格式为 ``{name: InputSpec}`` 的 ``dict`` ，或者为 ``None`` . 默认值为 ``None`` .
-    - **labels** (InputSpec|list|None, 可选): ``network`` 的标签，可以是 ``InputSpec`` 的实例，或者是一个 ``InputSpec`` 的 ``list`` ,或者为 ``None``. 默认值为 ``None`` .
+    - **inputs** (InputSpec|list|dict|None，可选):  ``network`` 的输入，可以是 ``InputSpec`` 的实例，或者是一个 ``InputSpec`` 的 ``list`` ，或者是格式为 ``{name: InputSpec}`` 的 ``dict`` ，或者为 ``None`` . 默认值为 ``None`` .
+    - **labels** (InputSpec|list|None，可选): ``network`` 的标签，可以是 ``InputSpec`` 的实例，或者是一个 ``InputSpec`` 的 ``list`` ，或者为 ``None``. 默认值为 ``None`` .
```
Comment (Ligoml) [inline docs/api/paddle/Model_cn.rst]: `:` --> `-`
`.` --> `。`

File: docs/api/paddle/Model_cn.rst [hunk 2]
```diff
@@ -13,12 +13,12 @@ Model
...
       
 .. note::
 
-    在动态图中， ``inputs`` 和 ``labels`` 都可以设置为 ``None``. 但是，在静态图中， ``input`` 不能设置为 ``None`` . 而如果损失函数需要标签(label)作为输入，则必须设置 ``labels`` ，否则，可以为 ``None`` 。
+    在动态图中，``inputs`` 和 ``labels`` 都可以设置为 ``None``. 但是，在静态图中，``input`` 不能设置为 ``None`` . 而如果损失函数需要标签(label)作为输入，则必须设置 ``labels`` ，否则，可以为 ``None`` 。
```
Comment (Ligoml) [inline docs/api/paddle/Model_cn.rst]: `.` --> `。`

File: docs/api/paddle/Tensor_cn.rst [hunk 3]
```diff
@@ -986,7 +986,7 @@ fill_diagonal_tensor(x, y, offset=0, dim1=0, dim2=1, name=None)
 :::::::::
 将输入Tensor y填充到Tensor x的以dim1、dim2所指定对角线维度作为最后一个维度的局部子Tensor中，输入Tensor x其余维度作为该局部子Tensor的shape中的前几个维度。
 其中输入Tensor y的维度要求是：最后一个维度与dim1、dim2指定的对角线维度相同，其余维度与输入Tensor x其余维度相同，且先后顺序一致。
-例如，有输入Tensor x，x.shape = (2,3,4,5)时, 若dim1=2，dim2=3，则y.shape=(2,3,4); 若dim1=1，dim2=2，则y.shape=(2,5,3); 
+例如，有输入Tensor x，x.shape = (2,3,4,5)时，若dim1=2，dim2=3，则y.shape=(2,3,4); 若dim1=1，dim2=2，则y.shape=(2,5,3);
```
Comment (Ligoml) [inline docs/api/paddle/Tensor_cn.rst:989]: `;` --> `；`

File: docs/api/paddle/all_cn.rst [hunk 4]
```diff
@@ -12,7 +12,7 @@ all
     - **x** （Tensor）- 输入变量为多维Tensor，数据类型为bool。
     - **axis** （int | list | tuple ，可选）- 计算逻辑与运算的维度。如果为None，则计算所有元素的逻辑与并返回包含单个元素的Tensor变量，否则必须在  :math:`[−rank(x),rank(x)]` 范围内。如果 :math:`axis [i] <0` ，则维度将变为 :math:`rank+axis[i]` ，默认值为None。
     - **keepdim** (bool，可选) - 是否在输出 Tensor 中保留减小的维度。除非 keepdim 为 True，否则输出 Tensor 的维度将比输入 Tensor 小一维，默认值为 False。
-    - **name** （str ， 可选）- 具体用法请参见 :ref:`api_guide_Name` ，一般无需设置，默认值为None。
+    - **name** （str ，可选）- 具体用法请参见 :ref:`api_guide_Name` ，一般无需设置，默认值为None。
```
Comment (Ligoml) [inline docs/api/paddle/all_cn.rst]: str后面没有空格

Comment (yang131313) [reply docs/api/paddle/all_cn.rst]: done

File: docs/api/paddle/Tensor_cn.rst (hunk 3 above)
Comment (yang131313) [reply docs/api/paddle/Tensor_cn.rst:989]: done

File: docs/api/paddle/Model_cn.rst (hunk 2 above)
Comment (yang131313) [reply docs/api/paddle/Model_cn.rst]: done

File: docs/api/paddle/Model_cn.rst (hunk 1 above)
Comment (yang131313) [reply docs/api/paddle/Model_cn.rst]: 参数后面的：还没改，句号done

**Pull request closed without merging**